feature-group-definitions/trusted-types.yml

```diff
@@ -1,4 +1,5 @@
 name: Trusted types
+description: The Trusted Types API is used to lock down insecure parts of the DOM API, to prevent client-side cross-site scripting (XSS) attacks.
```

Comment:
Trusted types feels too embedded into lots of parts of the platform to be called an API on its own.

```suggestion
description: Trusted types allow you to lock down insecure parts of the DOM API and prevent client-side cross-site scripting (XSS) attacks.
```